.gitignore

```diff
@@ -14,3 +14,4 @@
 **/mock_framework.zig
 **/*.ramdisk
 **/*.img
+zig-out/
```

Comment:
Shouldn't this go under the `# Zig ignore` section

Reply:
It should :) I have a shell alias that just appends what I want to ignore to the end of the file